debian/changelog

```diff
@@ -1,3 +1,36 @@
+snapcraft (2.2) xenial; urgency=medium
+
+  [ Leo Arias ]
+  * Add the unit tests to the autopkgtest suite. (LP: #1541421)
+  * Use docopt to handle example tests arguments. (LP: #1533395)
+  * integration-tests: add login and logout tests (LP: #1541391)
+  * move image into docs/images (LP: #1542422)
```

Comment:
Anything else? Not sure how this happened; but there were like 3 or 4 @elopio's around, so I might have confused this :wink: